system/handlers/admin/DataManager.cfc

```diff
@@ -344,6 +344,76 @@
 			);
 		</cfscript>
 	</cffunction>
+	<cffunction name="batchEditField" access="public" returntype="void" output="false">
+		<cfargument name="event"             type="any"     required="true" />
+		<cfargument name="rc"                type="struct"  required="true" />
+		<cfargument name="prc"               type="struct"  required="true" />
+		<cfscript>
+			var objectName   = rc.object;
+			var field        = rc.pickField ?: "";
+			fieldAttributes  = presideObjectService.getObjectProperty( objectName, field);
+			fieldType        = presideObjectService.getDefaultFormControlForPropertyAttributes( argumentCollection = 																									fieldAttributes);
+			formControl.name          = field;
+			formControl.label         = field;
+			formControl.maxlength     = fieldAttributes.maxlength ?: "";
+			formControl.minlength     = fieldAttributes.minlength ?: "";
+			formControl.object        = fieldAttributes.relatedto;
+			formControl.sourceObject  = fieldAttributes.relatedto;
+			formControl.relatedto     = fieldAttributes.relatedto;
+			formControl.type          = fieldType;
+			formControl.maxValue      = fieldAttributes.maxvalue  ?: "";
+			formControl.minValue      = fieldAttributes.minvalue  ?: "";
+			rc.renderObject           = renderFormControl( argumentCollection = formControl );
+			if(fieldType == "manyToManySelect") {
+				rc.renderSwitch           = renderFormControl( type  = "select",
+															name     = "overwrite",
+															label    = "Multi Edit Behaviour",
+															values   = [ "append", "overwrite" ], 
+															labels   = [ translateResource( uri="cms:datamanager.multiDataAppend.title" ),	   translateResource( uri="cms:datamanager.multiDataOverwrite.title" ) 
+																	  ] );
+			}
+			_addObjectNameBreadCrumb( event, objectName );
+			event.addAdminBreadCrumb(
+				  title = translateResource( uri="cms:datamanager.editrecord.breadcrumb.title" )
+				, link  = ""
+			);
+			event.setView( view="/admin/datamanager/batchEditField");
+		</cfscript>
+	</cffunction>
+
+	<cffunction name="batchEditAction" access="public" returntype="void" output="false">
+		<cfargument name="event"             type="any"     required="true" />
+		<cfargument name="rc"                type="struct"  required="true" />
+		<cfargument name="prc"               type="struct"  required="true" />
+		<cfscript>
+			rc.updateValue           = rc[rc.updateField];
```

Comment:
Feedback on code block below:

* `var` scope local variables
* Do not use `rc` for variables that are calculated by code. `rc` should be reserved for user input only. Use either local `var` scope, for variables that are only needed within the function, or `prc` for variables that need to be accessed later in views, etc.
* Do not assume variables exist, e.g. `rc.updateField`. Instead:

```
var updateField = rc.updateField ?: "";
// etc.
```